routes/api-routes.js

```diff
@@ -1,5 +1,14 @@
+const controllers = require('../controllers/index');
 const router = require('express').Router();
 
 router.get('/', (req, res) => res.json('Sample API get endpoint'));
```

Comment:
res.json('Sample API get endpoint')); : what is the end point we are looking at?